charts/signoz/templates/schema-migrator/migrations-upgrade.yaml

```
@@ -37,9 +37,11 @@ spec:
         - name: schema-migrator
           image: {{ include "schemaMigrator.image" . }}
           imagePullPolicy: {{ .Values.schemaMigrator.image.pullPolicy }}
+          env:
+            {{- include "snippet.clickhouse-credentials" . | nindent 12 }}
           args:
             - "--dsn"
-            - {{ include "schemamigrator.dsn" . | quote }}
+            - "tcp://{{ include "schemamigrator.url" $ }}?username=$(CLICKHOUSE_USER)&password=$(CLICKHOUSE_PASSWORD)"
```

It could be `.` instead of `$` here.

```suggestion
            - "tcp://{{ include "schemamigrator.url" . }}?username=$(CLICKHOUSE_USER)&password=$(CLICKHOUSE_PASSWORD)"
```